lib/storage/providers/MemoryOnlyProvider.js

```diff
@@ -0,0 +1,155 @@
+import _ from 'underscore';
+import sizeof from 'object-sizeof';
+import utils from '../../utils';
+
+// eslint-disable-next-line import/no-mutable-exports
+let store = {};
+
+const setInternal = (key, value) => {
+    store[key] = value;
+    return Promise.resolve(value);
+};
+
+const isJestRunning = typeof jest !== 'undefined';
+const set = isJestRunning ? jest.fn(setInternal) : setInternal;
+
+const provider = {
+    /**
+     * The name of the provider that can be printed to the logs
+     */
+    name: 'MemoryOnlyProvider',
+
+    /**
+     * Initializes the storage provider
+     */
+    init() {},
+
+    /**
+     * Get the value of a given key or return `null` if it's not available in memory
+     * @param {String} key
+     * @return {Promise<*>}
+     */
+    getItem(key) {
+        return Promise.resolve(store[key]);
```

Comment:
`store[key]` will return `undefined` for a missing key. We should coerce that to `null` like we do in the IDB provider, right?

Reply:
yes, indeed!